sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionState.scala

```diff
@@ -62,15 +62,17 @@ private[hive] class HiveSessionState(sparkSession: SparkSession)
       override val extendedResolutionRules =
         catalog.ParquetConversions ::
         catalog.OrcConversions ::
```

Comment:
These two rules need `MetastoreRelation`. Ideally, they should be after the rule `FindHiveSerdeTable`. 

I am fine to keep it if we plan to move it into optimizer rules.